docs/src/dictionary/en-custom.txt

```diff
@@ -8,24 +8,29 @@ Gitter
 MERCHANTABILITY
 MkDocs
 NONINFRINGEMENT
+NUL
 Preprocess
 PyPI
 TODO
 Tox
 Twemoji
 Virtualenv
+Willmer
 accessor
 deprecations
 expander
 globbing
 incrementer
 iterables
 linter
+nocover
```

Comment:
These are normally needed. They should be ignored automatically due to the PySpelling config.

Reply:
To clarify, both `nocover` and `pragma` will be ignored.

Reply:
```console
➜  bracex git:(master) python3 -m pyspelling -n python
Spelling check passed :)
```

Notice I already have `# pragma: no cover` in source: https://github.com/facelessuser/bracex/blob/master/bracex/__meta__.py#L134

Reply:
Use `# pragma: no cover`. That's why it failed.